go.mod

```diff
@@ -1,10 +1,9 @@
 module github.com/pion/ice/v2
 
-go 1.13
+go 1.23
```

Comment:
I intentionally bumped this. Turns out the declared version controls behavior such as whether or not the vendor directory is to be used.